core/embed/trezorhal/haptic.h

```diff
@@ -23,4 +23,6 @@ bool haptic_test(uint16_t duration_ms);
 // Play haptic effect
 void haptic_play(haptic_effect_t effect);
 
+void haptic_set_enabled(bool enable);
```

Comment:
See above for explanation why not include `feedback` here. 

I think `set_enabled` is better here - when we implement the set_intensity function i think its better to have clearer distinction what is actually being set. 

I renamed the `haptic_set` in https://github.com/trezor/trezor-firmware/commit/4f700c68cdc9f0452555bf40a49023c4b64f7614 so that we just have consisten, more descriptive names everywhere